<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibWindow</class>
 <widget class="QMainWindow" name="LibWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>961</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LibWindow</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::GroupedDragging</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="placeholderText">
         <string>Type something</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>978</width>
     <height>38</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLibrarian">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewLib"/>
    <addaction name="actionOpenLib"/>
    <addaction name="actionCloseLib"/>
    <addaction name="separator"/>
    <addaction name="actionDeleteLib"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Draw</string>
    </property>
    <addaction name="actionRectangle"/>
    <addaction name="actionLine"/>
    <addaction name="actionText"/>
    <addaction name="actionCircle"/>
    <addaction name="actionPolygon"/>
    <addaction name="actionPolyline"/>
    <addaction name="actionArrow"/>
    <addaction name="separator"/>
    <addaction name="actionuPin"/>
   </widget>
   <widget class="QMenu" name="menuSymbols">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionMove"/>
    <addaction name="actionRotate"/>
    <addaction name="actionStyle"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuHelp_2">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuLibrary">
    <property name="title">
     <string>Library</string>
    </property>
    <addaction name="actionSymbol"/>
    <addaction name="actionShape"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuLibrarian"/>
   <addaction name="menuLibrary"/>
   <addaction name="menuSymbols"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp_2"/>
  </widget>
  <widget class="QDockWidget" name="dwLibItems">
   <property name="windowTitle">
    <string>Library Items</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTreeView" name="tvLibItems"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pbNewShape">
         <property name="text">
          <string>New Shape</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbNewSymbol">
         <property name="text">
          <string>New Symbol</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbDeleteItem">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>50</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionNewLib"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="windowTitle">
    <string>Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="nameLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="addressLabel">
           <property name="text">
            <string>Address</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="addressLineEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="dateLabel">
           <property name="text">
            <string>Date</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDateEdit" name="dateDateEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_4">
   <property name="windowTitle">
    <string>Another</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNewLib">
   <property name="icon">
    <iconset>
     <normaloff>icons/png/32x32/add_to_folder.png</normaloff>icons/png/32x32/add_to_folder.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Create new library</string>
   </property>
  </action>
  <action name="actionCloseLib">
   <property name="text">
    <string>Close lib</string>
   </property>
   <property name="toolTip">
    <string>Close library</string>
   </property>
  </action>
  <action name="actionDeleteLib">
   <property name="text">
    <string>Delete Lib</string>
   </property>
   <property name="toolTip">
    <string>Delete library</string>
   </property>
  </action>
  <action name="actionSymbol">
   <property name="text">
    <string>New Symbol</string>
   </property>
   <property name="toolTip">
    <string>Create new symbol in library</string>
   </property>
  </action>
  <action name="actionShape">
   <property name="text">
    <string>New Shape</string>
   </property>
   <property name="toolTip">
    <string>Create new shape in library</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="text">
    <string>Line</string>
   </property>
  </action>
  <action name="actionText">
   <property name="text">
    <string>Text</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="text">
    <string>Circle</string>
   </property>
  </action>
  <action name="actionuPin">
   <property name="text">
    <string>Pin</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="text">
    <string>Move</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="text">
    <string>Rotate</string>
   </property>
  </action>
  <action name="actionArrow">
   <property name="text">
    <string>Arrow</string>
   </property>
  </action>
  <action name="actionPolygon">
   <property name="text">
    <string>Polygon</string>
   </property>
  </action>
  <action name="actionPolyline">
   <property name="text">
    <string>Polyline</string>
   </property>
  </action>
  <action name="actionStyle">
   <property name="text">
    <string>Style</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionOpenLib">
   <property name="text">
    <string>Open </string>
   </property>
   <property name="toolTip">
    <string>Open library</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
